Makefile

```diff
@@ -140,7 +140,7 @@ tests: check-tox ## Run tox -e unit against code
 	tox -e py3-unit
 
 .PHONY: verify
-verify: check-tox ## Run tox -e fmt,lint,spellcheck against code
+verify: check-tox ## Run tox -e ruff,fastlint,spellcheck against code
```

Comment:
I'd argue such a comment is useless and should be removed instead of trying to keep it in sync with the implementation of the target.

Reply:
Honestly, a fair point. I may come up with some new text altogether.

Reply:
+1 on @booxter's point.